pkgs/by-name/lo/lolcommits/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  bundlerEnv,
+  ruby,
+  buildRubyGem,
+  bundlerUpdateScript,
+  mplayer,
+  imagemagick,
+}:
+let
+  pname = "lolcommits";
+  version = "0.17.2";
+
+  deps = bundlerEnv rec {
+    inherit ruby version;
+    name = "${pname}-deps-${version}";
+    gemdir = ./.;
+  };
+
+  path = lib.makeBinPath [
+    mplayer
+    imagemagick
+  ];
+in
+buildRubyGem rec {
+  inherit pname version ruby;
+  gemName = pname;
+  source.sha256 = "sha256-Kwp8wZUn+cfrwEHRt11cNrfhjwfWJQbjw2ba9arIgLY=";
+
+  propagatedBuildInputs = [ deps ];
+
+  preFixup = ''
+    wrapProgram $out/bin/lolcommits --prefix PATH : ${path}
```

Comment:
```suggestion
    wrapProgram $out/bin/lolcommits \
      --suffix PATH : ${lib.makeBinPath [ mplayer imagemagick ]}
```

Reply:
Your proposed change isn't nixfmt-compliant (it would produce this instead):
```nix
  preFixup = ''
    wrapProgram $out/bin/lolcommits \
      --prefix PATH : ${
        lib.makeBinPath [
          mplayer
          imagemagick
        ]
      }
  '';
```
Which honestly IMO is uglier. Then again, this is a really minor point that's not really worth fighting over